gulp/scripts.js

```diff
@@ -3,7 +3,7 @@
 module.exports = function (gulp, gutil) {
   var scripts = require('../lib/scripts.js')(gulp, gutil);
 
-  gulp.task('scripts', ['scripts-main', 'scripts-tpls', 'scripts-deps', 'scripts-deps-metadata', 'scripts-activecast']);
+  gulp.task('scripts', ['scripts-main', 'scripts-tpls', 'scripts-deps', 'scripts-deps-metadata', 'scripts-activecast', 'scripts-activecast-tracker']);
```

Comment:
I suggest creating the following three tasks:

`scripts-activecast` (a task that just calls the other two activecast tasks)
`scripts-activecast-widget` (builds the activecast widget bundle)
`scripts-activecast-tracker` (builds the activecast tracker bundle)